include/vrv/linkinginterface.h

```diff
@@ -70,23 +70,23 @@ class LinkingInterface : public Interface, public AttLinking {
      * We have functor in the interface for avoiding code duplication in each implementation class.
      * Since we are in an interface, we need to pass the  Object (implementation) to
      * the functor method. These not called by the Process/Call loop but by the implementaion
-     * classes explicitely. See FloatingObject::FillStaffCurrentTimeSpanning for an example.
+     * classes explicitely. See FloatingObject::PrepareStaffCurrentTimeSpanning for an example.
      */
 
     /**
-     * See Object::FillStaffCurrentTimeSpanning
+     * See Object::PrepareStaffCurrentTimeSpanning
      */
-    virtual int InterfaceFillStaffCurrentTimeSpanning(FunctorParams *functorParams, Object *object);
+    virtual int InterfacePrepareStaffCurrentTimeSpanning(FunctorParams *functorParams, Object *object);
```

Comment:
Most of the renamed functions start with a verb -- "Calc", "Init", "Prepare", etc. This one starts with a noun. 

I'm not 100% sure what else to call it, but thought I might flag this as standing out.

Reply:
That is because this is not an Object functor but instead a Interface method redirected from a functor. A bit complicated to explain, but they are all prefixed with `Interface` to make that clear. Should be fine as is.